app/services/batch_invite_job.rb

```diff
@@ -20,19 +20,20 @@ def tell_devise_if_okay_to_resend_invitations
   end
 
   def invite_users_and_collate_results
-    results = @invites.flat_map do |organisation_id, email|
-      [organisation_id, result_of_inviting(invite_user(email, organisation_id))]
+    results = {}
+    @invites.each do |organisation_id, email|
```

Comment:
maybe could use inject here - but that's not urgent - could be a refactoring ticket for a beginner to work on